### PR TITLE
Added support for aws alb ingress controller

### PR DESCRIPTION
**Pull request by phenixdotnet**
Hi.
Please find a proposition to support [AWS Load balancer controller](https://github.com/kubernetes-sigs/aws-load-balancer-controller) directly from the chart configuration.

The AWS LB support is enabled by setting `regexPathStyle` to `aws-alb`

### CODE REVIEW

File: sentry/templates/ingress.yaml [hunk 1]
```diff
@@ -22,6 +22,20 @@ spec:
             backend:
               serviceName: {{ template "sentry.fullname" . }}-nginx
               servicePort: {{ .Values.nginx.service.port }}
+    {{- else }}
```
Comment (Mokto) [inline sentry/templates/ingress.yaml]: I think using `else if` would be easier to read: https://helm.sh/docs/chart_template_guide/control_structures/#ifelse

**Review by Mokto — CHANGES_REQUESTED** [review]

File: sentry/templates/ingress.yaml (hunk 1 above)
Comment (phenixdotnet) [reply sentry/templates/ingress.yaml]: okay no prob :)

File: sentry/templates/ingress.yaml [hunk 2]
```diff
@@ -22,6 +22,19 @@ spec:
             backend:
               serviceName: {{ template "sentry.fullname" . }}-nginx
               servicePort: {{ .Values.nginx.service.port }}
+    {{- else if eq (default "nginx" .Values.ingress.regexPathStyle) "aws-alb" }}
```
Comment (joel-schaal) [inline sentry/templates/ingress.yaml:25]: Wouldn't that be suitable for all cases (alb, traeffic,..) ?
I think the paths defined in that order remove the need for any RegEx.

Comment (phenixdotnet) [reply sentry/templates/ingress.yaml:25]: Not sure about how other ingress controller process the paths.
AWS alb is very ... limited, it process it in the configuration order but if nginx or traefik process it from more restrictive to less restrictive, this *could* not work as expected. 
And the main problem is still here: Each ingress controller has it's own regex syntax so anyway we have to deal with it.

File: sentry/templates/ingress.yaml [hunk 3]
```diff
@@ -40,6 +53,7 @@ spec:
               serviceName: {{ template "sentry.fullname" . }}-web
               servicePort: {{ .Values.service.externalPort }}
     {{- end }}
+    {{- end }}
```
Comment (Mokto) [inline sentry/templates/ingress.yaml]: This is extra right ?

**Review by Mokto — CHANGES_REQUESTED** [review]

File: sentry/templates/ingress.yaml (hunk 3 above)
Comment (phenixdotnet) [reply sentry/templates/ingress.yaml]: Yes you're right ^^'

**Review by Mokto — APPROVED** [review]

**Comment by phenixdotnet** [comment]
Thanks !

**Pull request merged**